protocol/activity_center.go

```diff
@@ -37,6 +37,8 @@ const (
 	ActivityCenterMembershipStatusPending
 	ActivityCenterMembershipStatusAccepted
 	ActivityCenterMembershipStatusDeclined
+	ActivityCenterMembershipStatusAcceptedPending
+	ActivityCenterMembershipStatusDeclinedPending
```

Comment:
@status-im/desktop-devs these are new states that desktop UI should be aware of

Reply:
How we should show those states in the UI?

Reply:
Everything related to UI is discussed here: https://github.com/status-im/status-desktop/issues/11649

Reply:
Should it be `...AcceptPending` and `..DeclinePending` (without `ed`)?
Or am I misunderstanding smth?

Reply:
Well the state is in "accepted pending" meaning, an admin has accepted the request but the decision is still pending until a control node confirms. Not sure if `-ed` is an issue here